plugins/generic/externalFeed/classes/ExternalFeedDAO.inc.php

```diff
@@ -176,7 +187,7 @@ function updateExternalFeed(&$externalFeed) {
 	 * @param $externalFeed ExternalFeed 
 	 * @return boolean
 	 */
-	function deleteExternalFeed($externalFeed) {
+	public function deleteExternalFeed($externalFeed) {
```

Comment:
This still uses the old conventions (https://github.com/pkp/ojs/pull/1115#discussion_r162740795).